start-utils

```diff
@@ -3,7 +3,7 @@
 function isURL {
   local value=$1
 
-  if [[ ${value:0:8} == "https://" || ${value:0:7} = "http://" ]]; then
+  if [[ ${value:0:8} == "https://" || ${value:0:7} == "http://" ]]; then
```

Comment:
I wonder if this quietly worked since most sites are https now.

Reply:
I ran some tests on it and the old way worked too... I [looked it up](https://unix.stackexchange.com/a/72041/1842) and apparently both ways work - I learned something today.

Probably good to be consistent I guess.

Reply:
That's good actually, because that was the same thing I was understanding from my man-page reading.